src/test/java/kg/jarkyn/server/Resource/ResourceReaderTest.java

```diff
@@ -0,0 +1,22 @@
+package kg.jarkyn.server.Resource;
+
+import kg.jarkyn.server.FileHelpers;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class ResourceReaderTest {
+
+    @Test
+    public void returnsFileByteContent() {
+        byte[] expectedContent = FileHelpers.plainFileByteContent();
+
+        assertArrayEquals(expectedContent, ResourceReader.readFile(FileHelpers.plainFileFullPath));
+    }
+
+    @Test
+    public void listsFolderContents() {
+        assertEquals(FileHelpers.publicFolderListing, ResourceReader.readFolder(FileHelpers.publicFolderPath));
```

Comment:
I think I know what you mean but I wouldn't say so
ResourceReader still has to look at the folder and do the work to return the contents

from `FileHelpers`:
```
public static String publicFolderPath = projectFolderPath + "/src/test/resources/public";
public static List<String> publicFolderListing = Arrays.asList("file1", "file2");
```

what do you think?